src/rdkafka_transport.c

```diff
@@ -694,7 +822,16 @@ static int rd_kafka_transport_ssl_connect (rd_kafka_broker_t *rkb,
 		goto fail;
 #endif
 
-        rd_kafka_transport_ssl_clear_error(rktrans);
+    if (rktrans->rktrans_rkb->rkb_rk->rk_conf.ssl.handshake_info) {
```

Comment:
This should be made into standard debug.
If it isn't too noisy (like up to 10 lines per SSL handshake) then it can use the `security` context, otherwise we should add a `sslhandshake` debug context - if there is value to it.
I believe one can use the `openssl s_client ..` to debug SSL issues just as well?

Reply:
OK in this case i will remove it.  I found it helpful during debugging but but using openssl s_client would probably be just as good.